.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -111,6 +111,13 @@ public override ParseResult Parse(ExtractResult extResult)
                 }
             }
 
+            // Japanese determine the subtype
+            if (Config.CultureInfo.Name == "ja-JP" && ret != null)
```

Comment:
Minor: This should be changed in the future to check just for the "ja" prefix, as "ja-*" could also be acceptable.

Reply:
Sure